### ISSUE TITLE
Add extraVolumes and extraVolumesMount in the chart

### ISSUE THREAD

**Issue by thibaultserti** [robusta-dev/robusta#1004]
**Is your feature request related to a problem?**
I want to use CSI secrets to create the K8S secret when the robusta runner is created. For this I need to add a volume and a volumeMounts as this:

```yaml
volumeMounts:
- mountPath: /mnt/secrets-store
    name: secrets-store-inline
volumes:
- name: secrets-store-inline
    csi:
    driver: secrets-store.csi.k8s.io
    readOnly: true
    volumeAttributes:
        secretProviderClass: <secret-provider-class-name>
```

**Describe the solution you'd like**
I want to have a value in the Chart to add volumes and volumesMounts.

**Comment by arikalon1** [comment]
Thanks for reporting it @thibaultserti 

You'd like to read these secrets from a Robusta playbook?
Can you please elaborate some more on the need for it?

**Comment by thibaultserti** [comment]
Thanks for the quick answer :

Precisely, I am using ArgoCD to provision all my apps and I use CSI to retrieve and sync the secrets from Vault.
To manage the lifecycle of the K8S secret, I need to use a CSI volume.

I just want to follow what is written here :
https://docs.robusta.dev/master/setup-robusta/configuration-secrets.html#managing-secrets

Fow now it's not really for robusta playbooks but more for robusta configuration.

**Comment by arikalon1** [comment]
Hi @thibaultserti 

Can you confirm this requirement is fixed here:
https://github.com/robusta-dev/robusta/pull/1074/files

**Comment by thibaultserti** [comment]
Yes it is, I close

**Comment by arikalon1** [comment]
Thanks @thibaultserti 

It will be released in 0.10.26 (should be within a week or so)
